apps/web/src/utils/formatEthPrice.ts

```diff
@@ -0,0 +1,22 @@
+import { formatEther, parseEther } from 'viem';
+
+export function formatEthPrice(price?: bigint): number | '...' {
+  if (price === undefined) {
+    return '...';
+  }
+  const value = parseFloat(formatEther(price));
+  if (value < 0.001) {
+    return parseFloat(value.toFixed(4));
+  } else {
+    return parseFloat(value.toFixed(3));
+  }
+}
```

Comment:
double check we don't have another util for this already

Reply:
i can confirm that we did not already have one.